src/style/app.module.css

```diff
@@ -514,26 +512,35 @@
 
 .searchButton {
   margin-bottom: 10px;
-  background-color: var(--search-button-bg);
-  border-color: var(--search-button-bg);
+  background-color: var(--search-button-bg) !important;
+  border-color: var(--search-button-bg) !important;
   position: absolute;
   z-index: 10;
   bottom: 0;
   right: 0;
   display: flex;
   justify-content: center;
   align-items: center;
+  transition:
+    box-shadow 0.2s ease,
+    transform 0.2s ease;
 }
 
 .searchButton:hover {
-  background-color: var(--search-button-hover-bg, var(--light-blue-color));
-  border-color: var(--search-button-hover-bg, var(--light-blue-color));
+  background-color: var(--search-button-bg) !important;
+  border-color: var(--search-button-bg) !important;
+  box-shadow: 0px 4px 10px rgba(0, 0, 0, 0.1);
```

Comment:
Use a variable name not the hex/rgba code. Ensure that the variable name you choose describes the function of the color and not the color itself. This will help with future attempts to having Dark Mode.
- For example `--search-button-bg` is much better than `--light-blue`